docs/src/main/asciidoc/security-openid-connect-providers.adoc

```diff
@@ -49,6 +49,8 @@ quarkus.oidc.client-id=<Client ID>
 quarkus.oidc.credentials.secret=<Secret>
 ----
 
+TIP: You can also use GitHub provider with `quarkus.oidc.application-type=service`, just set `quarkus.oidc.verify-access-token-with-user-info` configuration property to `true`.
+
```

Comment:
Interesting the new rules picked up `In order to` in the follow up section, but I think we should handle it in another PR as I guess most of the docs can be optimized